src/plugins/data_source/config.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { schema, TypeOf } from '@osd/config-schema';
+
+export const configSchema = schema.object({
+  enabled: schema.boolean({ defaultValue: false }),
+  wrappingKeyName: schema.string({
+    minLength: 1,
+    maxLength: 100,
```

Comment:
where is this maxLength come from?

Reply:
> where is this maxLength come from?

There are no explicit restrictions in the public doc or codebase. It's just a dummy length based on the assumption that it should remain short.